src/main/java/frc/robot/Constants.java

```diff
@@ -8,16 +8,24 @@
 
     public static int CLIMB_TALON = 6;
 
-    public static double LEFT_MASTER_FF = 0;
-    public static double LEFT_MASTER_P = 0;
+    public static double LEFT_MASTER_FF = 0.2;
+    public static double LEFT_MASTER_P = 0.2;
     public static double LEFT_MASTER_I = 0;
     public static double LEFT_MASTER_D = 0;
 
-    public static double RIGHT_MASTER_FF = 0;
-    public static double RIGHT_MASTER_P = 0;
+    public static int LEFT_MASTER_ACCELERATION = 0;
+    public static int LEFT_MASTER_VELOCITY = 0;
```

Comment:
These constants need to be set to something, for now lets try 3300 for both of them on each side